src/formatDate.js

```diff
@@ -50,7 +50,29 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldDateFormat = date.split(fromFormat[3]);
```

Comment:
It is not clear, what `fromFormat[3]` is referring to here and `toFormat[3]` on the 75th line. Probably, would be better to add extra variables for separators

Reply:
Yep, you rigth. Better name them)